### PR TITLE
 Feature: reset latched faults on command 

### PR DESCRIPTION
**Pull request by evanmayer**
Implements [Copley AN111: Handling Faults in CANopen and EtherCAT](https://www.servo.jp/member/admin/document_upload/AN111-Handling-Faults.pdf)

### CODE REVIEW

File: blast_config/command_list.c
Comment (brendal4) [inline blast_config/command_list.c:1]: Adds ability to send command to reset latch faults on any of the three motor controllers in the chain: reaction wheel, pivot, elevation axis.

File: mcp/motors/ec_motors.c [hunk 1]
```diff
@@ -1645,15 +1713,6 @@ static int motor_pdo_init(int m_periph)
     if (!ec_SDOwrite32(m_periph, ECAT_RXPDO_MAPPING, 2, map.val)) {
         blast_err("Failed mapping!");
     }
-    // map_pdo(&map, ECAT_LATCHED_DRIVE_FAULT, 32);
-    // if (!ec_SDOwrite32(m_periph, ECAT_RXPDO_MAPPING, 3, map.val)) {
-    //     blast_err("Failed mapping!");
-    // }
-    // Uncomment this once the phasing angle readout has been debugged
```
Comment (brendal4) [inline mcp/motors/ec_motors.c:1652]: Deleted because bug has been fixed and this is no longer necessary? I.e. why delete this instead of uncomment?

**Review by ianlowe13 — APPROVED** [review]

**Review by brendal4 — COMMENTED** [review]
See comment on ec_motors.c

File: mcp/motors/ec_motors.c (hunk 1 above)
Comment (evanmayer) [reply mcp/motors/ec_motors.c:1652]: This object (0x21C0) is not PDO-mappable. Phase init is decidedly a pre-operation step, so it doesn't make sense to try to map it as a process data object like other thing we'll need to read/write real-time. We set our phase initialization mode for each motor to use encoders only in CME...I'll have to see which mode that corresponds to here for documentation purposes, and maybe to update `rw_init_phasing` or something. This code isn't really valid. Fixed a bug below this though.

**Pull request merged**